### PR TITLE
Add descriptions to GraphQL schema entities

### PR DESCRIPTION
**Pull request by manumonti**
This PR solves #25

### CODE REVIEW

File: schema.graphql [hunk 1]
```diff
@@ -57,14 +73,16 @@ interface Delegation {
 }
 
 type StakeDelegation implements Delegation @entity {
-  # ID is the delegatee address
+  "StakeDelegation represents staked T tokens to which DAO's voting power has been delegated"
```
Comment (manumonti) [inline schema.graphql]: @r-czajkowski it would be great if you can review this description

File: schema.graphql [hunk 2]
```diff
@@ -57,14 +73,16 @@ interface Delegation {
...
+  "ID is delegatee ETH address"
   id: ID!
   totalWeight: BigInt!
   stakeDelegators: [StakeData!] @derivedFrom(field: "delegatee")
 }
 
 type TokenholderDelegation implements Delegation @entity {
-  # ID is the delegatee address
+  "StakeDelegation represents liquid T tokens to which DAO's voting power has been delegated"
```
Comment (manumonti) [inline schema.graphql]: @r-czajkowski and this.

File: schema.graphql [hunk 3]
```diff
@@ -57,14 +73,16 @@ interface Delegation {
...
+  "StakeDelegation represents staked T tokens to which DAO's voting power has been delegated"
+  "ID is delegatee ETH address"
   id: ID!
   totalWeight: BigInt!
   stakeDelegators: [StakeData!] @derivedFrom(field: "delegatee")
 }
 
 type TokenholderDelegation implements Delegation @entity {
-  # ID is the delegatee address
+  "TokenholderDelegation represents liquid T tokens to which DAO's voting power has been delegated"
```
Comment (manumonti) [inline schema.graphql]: @r-czajkowski and this

File: schema.graphql [hunk 4]
```diff
@@ -56,15 +71,17 @@ interface Delegation {
   totalWeight: BigInt!
 }
 
+"StakeDelegation represents staked T tokens to which DAO's voting power has been delegated"
```
Comment (manumonti) [inline schema.graphql]: @r-czajkowski it would be great if you can review this description

File: schema.graphql [hunk 5]
```diff
@@ -56,15 +71,17 @@ interface Delegation {
...
 type StakeDelegation implements Delegation @entity {
-  # ID is the delegatee address
+  "ID is delegatee ETH address"
   id: ID!
   totalWeight: BigInt!
   stakeDelegators: [StakeData!] @derivedFrom(field: "delegatee")
 }
 
+"TokenholderDelegation represents liquid T tokens to which DAO's voting power has been delegated"
```
Comment (manumonti) [inline schema.graphql]: @r-czajkowski and this

File: schema.graphql [hunk 6]
```diff
@@ -56,15 +71,17 @@ interface Delegation {
...
 type TokenholderDelegation implements Delegation @entity {
-  # ID is the delegatee address
+  "ID is delegatee ETH address"
   id: ID!
   # The voting weight from the snapshot mechanism in the token and staking
```
Comment (r-czajkowski) [inline schema.graphql]: Maybe we could change this comment to docs. I mean comment for `totalWeight` in lines 86-93.

Comment (manumonti) [reply schema.graphql]: Can you check it now?

**Review by r-czajkowski — APPROVED** [review]

**Pull request merged**